herwig.sh

```diff
@@ -1,11 +1,12 @@
 package: Herwig
 version: "%(tag_basename)s"
-tag: "v7.1.2-alice1"
-source: https://github.com/alisw/herwig
+tag: "alice/v7.2.0"
+source: https://github.com/mfasDa/herwig
```

Comment:
Ok for testing, but for the final deployment this should be changed to a tag (not a branch) from alisw.

Reply:
Of course, preparing the branch for Herwig 7.2.

Reply:
@mfasDa merged and tagged as v7.2.0